dotCMS/src/main/java/com/dotmarketing/beans/transform/MultiTreeTransformer.java

```diff
@@ -0,0 +1,49 @@
+package com.dotmarketing.beans.transform;
+
+import com.dotcms.util.transform.DBTransformer;
+import com.dotmarketing.beans.MultiTree;
+import com.dotmarketing.util.UtilMethods;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+
+public class MultiTreeTransformer implements DBTransformer<MultiTree> {
+
+    //MultiTree fields
+    private static final String CHILD = "child";
+    private static final String PARENT1 = "parent1";
+    private static final String PARENT2 = "parent2";
+    private static final String RELATION_TYPE = "relation_type";
+    private static final String TREE_ORDER = "tree_order";
+
+    private final List<MultiTree> list = new ArrayList<>();
+
+    @Override
+    public List<MultiTree> asList() {
+        return this.list;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "list". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)